Packs/Workday/Integrations/WorkdayIAMEventsGenerator/WorkdayIAMEventsGenerator.py

```diff
@@ -970,81 +827,158 @@
-    else:
-        integration_context = get_integration_context()
-        reports = NEW_HIRE_REPORT['Report_Entry']
-        for report in reports:
-            email = report.get('Email_Address')
-            if email == user_email:
-                integration_context['Report_Entry'].append(report)
-                set_integration_context(integration_context)
+def get_new_hire_reports(user_email, name):
+    integration_context = get_integration_context()
+    if not name or name == '':
+        return_error('Please insert the user full name (first name and last name).')
+    try:
+        first_name = name.split()[0]
+        last_name = name.split()[1]
+    except Exception:
+        return_error('Missing first or last name.')
+    report_entry = {
+        "Report_Entry": [
+            {
+                "Employee_Type": "Regular",
+                "Leadership": "No",
+                "Work_Country_Code": "840",
+                "Street_Address": "3000 Tannery Way",
+                "Employment_Status": "Active",
+                "VP_Flag": "N",
+                "Mgr_ID": "115069",
+                "Cost_Center_Description": "IoT - PM",
+                "GDPR_Country_Flag": "0",
+                "Director_Flag": "N",
+                "Email_-_Primary_Home": "test6@testing.com",
+                "First_Name": first_name,
+                "Last_Hire_Date": "06/15/2020",
+                "People_Manager_Flag": "N",
+                "Department": "Enterprise R&D:FWaaP",
+                "Workday_ID": "9aa7e309929e013ff3c6e3440803b833",
+                "Postal_Code": "95054",
+                "Rehired_Employee": "No",
+                "Org_Level_1": "All R&D",
+                "Org_Level_3": "FWaaP",
+                "Country_Name": "United States Of America",
+                "Org_Level_2": "Enterprise R&D",
+                "Emp_ID": "115074",
+                "Job_Family": "Product Management",
+                "Preferred_Name_-_First_Name": "Chester",
+                "Nikesh Arora": "False",
+                "Management_Level_1": "Nikesh Arora",
+                "Work_Country_Abbrev": "US",
+                "Management_Level_2": "Timmy Turner",
+                "Email_Address": user_email,
+                "Title": "Product Line Manager",
+                "City": "Santa Clara",
+                "Work_State_US_Only": "California",
+                "Job_Code": "5225",
+                "PAN_CF_Okta_Location_Region": "Americas",
+                "Last_Name": last_name,
+                "Job_Function": "Product Management Function",
+                "State": "California",
+                "Exec_Admin_Flag": "N",
+                "Preferred_Name": "Chester McBadbat",
+                "Regular_Employee_Flag": "Y",
+                "Preferred_Name_-_Last_Name": "McBadbat",
+                "Cost_Center_Code": "651116",
+                "Location": "Office - USA - CA - Headquarters"
+            }
```

Comment:
let's take this entry and put it as a constant outside the function (no need for the outer "Report_Entry") - for readability. then just modify and append it to the integration context.